### DISCUSSION TITLE
I have made this Template -Guide to make a Retype website for your CV/projects.

### DISCUSSION THREAD

**Discussion by luislgq** [retypeapp/retype#313] [Show and tell]
Hello!

I had the idea of making a website where I can share the projects I have been working on and my educational and work experience... Basically a companion website for my CV. And since I was doing it, I figured I could start by making a quick guide and template repo on how can people set their own site up:

https://github.com/luislgq/retypetemplatewebsite

Check it out, let me know if I am missing anything, or if the workflow is already out of date, have not been keeping up if there have been new patches or anything. 

Thank you!

**Comment by geoffreymcgill** [comment]
This is fantastic. Thanks for putting this guide together. 🙌 

I'm going to send you a pull-request with a few revisions. Retype v2 has been released, so the workflow file can be updated to build with that new release. I'll send you changes.

**Reply by geoffreymcgill** [reply]
Sent a PR with the suggested revisions, see 
https://github.com/luislgq/retypetemplatewebsite/pull/1

Hope this helps.

**Reply by luislgq** [reply]
Hello! I am glad you like it. Can you please let me know if there is any changelog of the new version? anything I should know? 


Feel free to share the repo with anyone who might need it !

**Reply by geoffreymcgill** [reply]
> Can you please let me know if there is any changelog of the new version?

https://retype.com/roadmap/

The release notes for each release are linked from the roadmap.

Hope this helps.
